docs/pages/button.js

```diff
@@ -127,58 +119,48 @@ card(
         type: `'sm' | 'md' | 'lg'`,
         required: false,
         defaultValue: 'md',
-        description:
-          'Display a button in different sizes. Size changes the component padding modifying its fixed height: sm (32px), md (40px), lg (48px).',
-        href: 'size',
+        description: ['sm: 32px, md: 40px, lg: 48px'],
+        href: 'Size',
       },
       {
         name: 'text',
         type: 'string',
         required: true,
         defaultValue: null,
         description: [
-          'Text to render inside the button to convey the function and purpose of the button. The button text has a fixed size.',
+          'Text to render inside the Button to convey the function and purpose of the Button. The Button text has a fixed size.',
```

Comment:
I'd move `The Button text has a fixed size` to the variant section (Size variant) We explain

Button is available in 3 fixed sizes: [...] The Button text has always a fixed size of xx pixels.

Reply:
Done!